PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Sentinel One Agent.md

```diff
@@ -0,0 +1,64 @@
+#### Name
+
+Windows - Install Sentinel One Agent| v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+$siteToken="SITE TOKEN"
+$installerURL="https://path/to/url.exe"
+
+############### Do Not Edit Below This Line ###############
+
+$installerTempLocation="C:\Windows\Temp\SentinelOneAgentInstaller.exe"
+
+if (Get-Service "SentinelOneService" -ErrorAction SilentlyContinue) {
+    Write-Host "Sentinel One Agent already installed, nothing to do."
+    exit 0
+}
+Write-Host "Sentinel One Agent not installed."
+
+Write-Host "Downloading Sentinel One Agent installer now."
+try {
+    Invoke-WebRequest -Uri $installerURL -OutFile $installerTempLocation
+}
+catch {
+    Write-Error "Unable to download Sentinel One Agent installer."
+    exit 1
+}
+Write-Host "Finished downloading Sentinel One Agent installer."
+
+Write-Host "Installing Sentinel One Agent now, this may take a few minutes."
+try {
+    ."$installerTempLocation" --dont_fail_on_config_preserving_failures -t $siteToken
+}
+catch {
+    Write-Error "Failed to run Sentinel One Agent installer."
+    exit 1
+}
+Write-Host "Sentinel One Agent installer returned $($installerProcess.ExitCode)."
```

Comment:
This looks good but when I ran this I did see the command results return the following:

![Screen Shot 2022-09-20 at 2 36 28 PM](https://user-images.githubusercontent.com/54448601/191359725-a381c3a0-f082-42e9-875a-e85e3851a423.png)

The text of the exit code 0 is not returned here